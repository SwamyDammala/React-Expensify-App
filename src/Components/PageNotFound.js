import React from 'react'
import { Link } from 'react-router-dom'

const PageNotFound=()=>(
    <div>
        404 Error Not found <Link to="/">Go Home</Link>
    </div>
)

export default PageNotFound