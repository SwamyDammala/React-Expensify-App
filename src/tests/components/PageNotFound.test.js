import React from 'react'
import {shallow} from 'enzyme'
import PageNotFound  from '../../Components/PageNotFound'


test('Should test Page not found',()=>{
    const wrapper=shallow(<PageNotFound />)
    expect(wrapper).toMatchSnapshot()
})