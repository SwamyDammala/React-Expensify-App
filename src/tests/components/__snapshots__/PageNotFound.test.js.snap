// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Page not found 1`] = `
<div>
  404 Error Not found 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
